### DISCUSSION TITLE
Warum [Error: Die SessionID stimmt nicht] ?

### DISCUSSION THREAD

**Discussion by ElbPsi** [fabiankachlock/zwoo#260] [Bug Reports]
Hallo Team,
beim erstellen eines neuen Spiels (öffentlich) erhielt ich die Fehlermeldung [Error: Die SessionID stimmt nicht]. Auch ohne Leerzeichen bekam ich die Fehlermeldung.
Bitte die Fehlermeldung erklären.

**Comment by fabiankachlock** [comment]
Hallo Elb,
vielen Dank für den Bug Report.

Der Fehler tritt auf, wenn keine Login Session besteht oder die aktuelle Login Session ungültig ist. Anscheinend wurde der Login Status fälschlicher Weiße als "Angemeldet" angezeigt, ein erneutes Anmelden sollte das Problem vorerst beheben.

zwoo Team

**Reply by fabiankachlock** [reply]
linked: #264; ZWOO-93
